guide/english/css/overflow/index.md

```diff
@@ -63,3 +63,4 @@ If the content overflows the Y-axis, then that content will be hidden, whilst a
 #### More Information:
 
 CSS-Tricks: <a href='https://css-tricks.com/almanac/properties/o/overflow/' target='_blank' rel='nofollow'>overflow</a>
+W3Schools: <a href='https://www.w3schools.com/css/css_overflow.asp' target='_blank' rel='nofollow'>image source</a>
```

Comment:
We tend to avoid adding links to w3schools.com as they sometimes have incorrect or incomplete information.